workshop/content/lab1.3_configure.md

```diff
@@ -74,18 +76,20 @@ imc-controller-5b75d458fc-ptvm2        1/1     Running   0          7m19s
 imc-dispatcher-64f6d5fccb-kkc4c        1/1     Running   0          7m18s
```

Comment:
I have more pods in my output.  I think it might be because of the new `KnativeKafka` resource.

My list:

```
eventing-controller-774cb8f795-mqlnt                       1/1     Running     0          63m
eventing-webhook-7d48698d57-6tnbh                          1/1     Running     0          63m
imc-controller-74677df7b9-4k6tq                            1/1     Running     0          63m
imc-dispatcher-6d7548bc78-7mjt9                            1/1     Running     0          63m
kafka-ch-controller-688656c68f-7fhkg                       1/1     Running     0          56m
kafka-ch-dispatcher-5589c8df9f-j2dms                       1/1     Running     0          56m
kafka-controller-manager-56bc769b5f-wtst2                  1/1     Running     0          56m
kafka-webhook-847cbdc68d-drcmn                             1/1     Running     0          56m
mt-broker-controller-6b8b787b97-49h27                      1/1     Running     0          63m
mt-broker-filter-5d6875d7c7-xbdz7                          1/1     Running     0          63m
mt-broker-ingress-787c4b9b76-kthxn                         1/1     Running     0          63m
storage-version-migration-eventing-eventing-0.18.6-nbtgr   0/1     Completed   0          63m
sugar-controller-54c7657b78-chdqn                          1/1     Running     0          63m
```

Reply:
You have the updated list in the kafka lab.

Reply:
Great catch

Reply:
@jkeam The old list is still here.  (lab1.3_configure)

Reply:
I'll work on this now.

Reply:
Updated!